### PR TITLE
feat: dim sun icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2022-03-28 at 19 34 25@2x |

### CODE REVIEW

**Comment by mittalyashu** [comment]
I also kind-of don't look like sun either 😅

**Comment by fdev** [comment]
What is the use case for this icon? The most common reason for the sun to be "dim" is when there are clouds in front of it, and there is [already an icon](https://lucide.dev/icon/cloud-sun) for that.

**Comment by delnyn** [comment]
yeah well I get the use case but would it be really useful?

**Comment by mittalyashu** [comment]
Every icon can have it's uses, someway or other 🤷

**Review by ericfennis — APPROVED** [review]

File: tags.json [hunk 1]
```diff
@@ -2286,6 +2286,10 @@
     "weather",
     "light"
   ],
+  "sun-dim": [
+    "dim sun",
+    "dim"
```
Comment (bduffany) [inline tags.json:2291]: mind adding "brightness" to the tags here? maybe also add "brightness" to the regular "sun" icon as well

File: tags.json [hunk 2]
```diff
@@ -2286,6 +2286,10 @@
     "weather",
     "light"
   ],
+  "sun-dim": [
+    "dim sun",
```
Comment (bduffany) [inline tags.json:2290]:
```suggestion
    "brightness",
```

(remove "dim sun" tag as it's not really useful, and add "brightness")

Comment (mittalyashu) [reply tags.json:2290]: Thanks 🙂

**Comment by karsa-mistmere** [comment]
How about something like this?
image

**Comment by mittalyashu** [comment]
> How about something like this? image

OMG! That's a good one.

//cc @ericfennis

**Comment by ericfennis** [comment]
@karsa-mistmere @mittalyashu I think this looks even better!

**Comment by karsa-mistmere** [comment]
I think there's maybe even space for multiple dim levels (I've just checked and iOS uses the same concept for its brightness level indicator icon).
image

I don't have access to the PR code, so here's a Figma file: https://www.figma.com/file/iRBudsI3NG79KtEODp1V5Z/lucide%2Fbrightness-dim-(%23545)?node-id=0%3A1

**Comment by delnyn** [comment]
`brightness-full` breaks guidelines, but everything else is great :0

**Comment by karsa-mistmere** [comment]
> `brightness-full` breaks guidelines, but everything else is great :0

Yeah, at that point due to some other icons I'd been under the impression that this particular guideline could be broken on a case by case basis. Let's scratch full and stick with medium and low (and potentially night light).

**Comment by delnyn** [comment]
yeah all the icons that break this rule are basically from the feather icon set and need to be refactored

**Comment by karsa-mistmere** [comment]
> yeah all the icons that break this rule are basically from the feather icon set and need to be refactored

Yeah, I've been on it this last week. 😁

**Review by ericfennis — CHANGES_REQUESTED** [review]
@mittalyashu I think the icons from @karsa-mistmere fit better. Do you want to change them?

**Comment by mittalyashu** [comment]
@ericfennis just added now 🙂

**Review by ericfennis — APPROVED** [review]

**Pull request merged**